src/lib/AST/BitcodeWriter.cpp

```diff
@@ -422,6 +426,12 @@ RecordsByBlock{
     {BI_FRIEND_BLOCK_ID,
         {FRIEND_SYMBOL}},
     // FriendInfo
+
+    // UsingInfo
+    {BI_USING_BLOCK_ID,
+        {USING_SYMBOLS, USING_CLASS}},
+
+    // EnumeratorInfo
```

Comment:
You still need an entry for `BI_ALIAS_BLOCK_ID` (it should look like the `BI_ENUMERATOR_BLOCK_ID` entry)